### DISCUSSION TITLE
Welcome to MsdialWorkbench Discussions!

### DISCUSSION THREAD

**Discussion by kozo2** [systemsomicslab/MsdialWorkbench#86] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Melogmo** [comment]
Hello Dr. Kozo
Thank you for the invitation on this GitHub discussions. 
I am Yanick Melogmo, PhD student in Biochemistry at University of Yaoundé 1, Cameroon. 
I am working on some Cameroonian medicinal herbs for their antibacterial (antishigellal) properties and the exploration of the likely mode of action of active ingredients (isolated compounds/subfractions). 
In this GitHub community, I would like to learn more about metabolomics and molecular networking tools for the establishment of metabolites profile of the active plants extracts. 

I am excited to be here and will expect that we will share our benefit experience.

**Reply by kozo2** [reply]
Hi Melogmo, (by the way I am not a Dr :))
You are the first commenter. Thanks!
Then move to #109 to discuss molecular networking!

**Comment by Soumya624** [comment]
Hi @kozo2 ,

Is there any dummy .wiff available for testing? I am using the current version of MS-Dial. But, couldn't find any demo folder for mass spectrometry data. Any link/reference of the archive in the README.md would be highly helpful!

Thanks!

**Reply by kozo2** [reply]
@Soumya624 

Sorry for the late reply.
I made that reply in https://github.com/systemsomicslab/MsdialWorkbench/issues/128 .

**Comment by niralmaruda** [comment]
Hi @kozo2 
I'm an enthusiastic software developer with 9 months of experience in C#. Your project piques my interest, and I'm eager to contribute. Could you please guide me on where I can start? I'm excited to be part of this team and make a positive impact with my skills.

**Comment by bdGut333ee** [comment]
Hi, 
I do untargeted multi omics to understand reactions during food processing. I am wondering when there will be a Linux version released. Thanks

**Reply by janikkokot** [reply]
Our lab would also be interested in a linux version.

I have a hard time tracking down the most recent linux release, since the homepage has moved, does anyone have a working link to a linux release?

**Comment by Tai-Tengsuttiwat** [comment]
Hello everyone,
I am using the MS-DIAL for my LC-MS project and found an error saying '**_Error is occured in GapFilling_**'. I am wondering what could be the reasons for this and how to fix it, hopefully. FYI, it went well when I ran its negative mode. Plus, I have tried running the positive mode for a few times, but it always get stuck at the GapFilling step.
Thank you so much 
Tai
![Error gap filling](https://github.com/user-attachments/assets/e74a414b-9c73-4dee-8d05-89dd7e51ad78)

**Comment by AymanSaber89** [comment]
Hello,

I recently started using MS-DIAL for LC-HRMS suspect screening and I am still a beginner with the software.

My current workflow involves screening environmental contaminants using an in-house suspect list and spectral library. I would like to better understand how to correctly perform peak integration, inspect MS/MS spectra, and apply in-silico fragmentation tools for compound identification.

Previously I worked mainly with MetaboScape and MetFrag, but now I would like to learn the MS-DIAL + MS-FINDER workflow more deeply.

Could you please recommend any tutorials, documentation, or training materials (videos, courses, or guides) that could help beginners become familiar with MS-DIAL for suspect screening and identification?

Thank you very much for developing and maintaining this excellent tool.
